build.rs

```diff
@@ -40,5 +48,60 @@ fn main() {
         println!("cargo:rustc-link-lib=static=chck-xdg");
         println!("cargo:rustc-link-lib=static=wlc-protos");
         println!("cargo:rustc-link-lib=static=wlc");
+    } else if let Err(s) = find() {
+        let _ = writeln!(io::stderr(), "{}", s);
+        process::exit(1);
+    }
+}
+
+fn find() -> Result<(), Error> {
+
+    let package_name = "wlc";
+    let shared_libs = ["wlc"];
+
+    // https://github.com/way-cooler/rust-wlc#rust-wlc currently
+    // (i.e. commit 97e0d5816fe7ec97dae20196d1e25c550f60f08f) suggests
+    // version 0.0.7.
+    let version = if cfg!(feature = "0.0.8") {
```

Comment:
Ok fair enough, it doesn't add that much more to the release schedule, and it ensures it's being compiled with the right version (which has been an issue in the past...). However, you should remove the fallback to searching for previous versions, since rust-wlc is not compatible with older version of libwlc. If it can't find the latest version (which can be specified in a constant, because again it makes no sense to have a feature flag if you can only use the lates version) then it should just panic with an error message saying the required version.